worker.go

```diff
@@ -104,13 +104,18 @@ func (opt *JobOptions) Validate() error {
 }
 
 var (
+	// ErrDoNotRetry is returned if the job should not be retried;
+	// this may be because the job is unrecoverable, or because
+	// the handler has already rescheduled it.
+	ErrDoNotRetry = errors.New("do not retry")
```

Comment:
need to add a prefix "work: " to be clear for package user that it is from this package